gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/artifacts/ExtractFileFromJar.java

```diff
@@ -33,7 +32,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@CacheableTransform
 public abstract class ExtractFileFromJar implements TransformAction<FileExtractParameter> {
```

Comment:
Might be worth commenting so we don't try to get clever and add it back. Great PR description, fantastic find!

Reply:
Good idea, have done so.

Reply:
you should use `org.gradle.work.DisableCachingByDefault`. It also allows you to include a reason as well. Should also prevent a regression if by any chance they turn on caching automatically.